src/annotator/config/url-from-link-tag.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Return the href of the first annotator link in the given
```

Comment:
I think the wording here could be more succinct. Something along the lines of:

```
Return the URL of a resource related to the Hypothesis client that has been stored in
the page via a `<link type="application/annotator+{type}">` tag.
```

Below that I think it would be useful to give some additional context about why this mechanism exists.

```
These link tags are generally written to the page by the boot script, which may be executed
in a separate JavaScript realm (eg. in the browser extension), and thus can share information
with the annotator code via the DOM but not JS globals.
```

I suggest the rest of the comment, which refers to specific uses of this function, can be removed as it is going to get out of date if new uses of this code are added elsewhere.